src/modules/application/application.controller.ts

```diff
@@ -40,8 +42,11 @@ export class ApplicationController {
     status: HttpStatus.OK,
     type: Application,
   })
-  public async getByOwner(@Param('owner') owner: string) {
-    return await this.applicationService.getByOwner(owner);
+  public async getByOwner(
+    @Param('owner') owner: string,
+    @Query('withRoles', BooleanPipe) withRoles: boolean
+  ) {
+    return await this.applicationService.getByOwner(owner, { withRoles });
```

Comment:
```suggestion
    return this.applicationService.getByOwner(owner, { withRoles });
```

Reply:
with nestJS you don't need to add the await when the return keyword will be executed last. It automatically resolves it.